apps/examples/mediaplayer/mediaplayer_main.cpp

```diff
@@ -321,22 +316,25 @@ class MediaPlayerController
 	void start()
 	{
 		while (true) {
-			auto test = selectSource();
-			if (test < 0) {
+			vector<string> sourceList = {"Exit APP", "Test PCM", "Test BUFFER", "Test HTTP"};
+			listDirEntries("/rom", sourceList);
```

Comment:
It seems that "sourceList" is loaded only once, isn't it..?

Reply:
sourceList can be changed, so it should be re-initialized

Reply:
Hmm.. it seems that all files under "/rom" are uploaded only once when flashing and no files can be created during runtime since "/rom" is read-only.
When and how sourceList can be changed..?